frontend/src/api/server/index.js

```diff
@@ -35,6 +36,10 @@ axios.interceptors.response.use(
   // 404 is considered an error
   error => {
     if (error.response) {
+      let path = '/restricted';
+      if (error.response.status === 401 || error.response.status === 403) {
+        router.replace(path);
+      }
```

Comment:
Do we want separate redirects for 401 (Unauthorized) vs 403 (Forbidden). e.g. 401 -> /login and 403 -> /restricted ?